anthem/lyrics/modules.py

```diff
@@ -41,3 +42,12 @@ def update_translations(ctx, module_list, overwrite=False):
     ctx.log_line("Reloading translations for %s" % str(module_list))
     mods.with_context(overwrite=overwrite)
     getattr(mods, method_name)()
+
+
+def nuke_translations(ctx, module_list):
+    """ Remove translations from module list"""
+    if not isinstance(module_list, list):
+        raise AnthemError(
+            u"You have to provide a list of " "module's name to remove the translations"
```

Comment:
second string should be on the second line, or merged in a single string.

Reply:
Merci